### DISCUSSION TITLE
zh/guide/media_server/playback_authentication

### DISCUSSION THREAD

**Discussion by giscus** [ZLMediaKit/docs#17] [Announcements]
# zh/guide/media_server/playback_authentication

使用 ZLMediaKit 的 MediaServer 进程可以实现播放鉴权： image

https://docs.zlmediakit.com/zh/guide/media_server/playback_authentication.html

**Comment by zhaowenduo** [comment]
May I ask if you can provide specific authentication methods and usage instructions for Hooks
